docs/source/jit_language_reference.rst

```diff
@@ -0,0 +1,438 @@
+     - Partially Supported
+     - Some keywords aren't reserved
+   * - `2.3.2. Reserved classes of identifiers <https://docs.python.org/3/reference/lexical_analysis.html#reserved-classes-of-identifiers>`_
+     - Partially Supported
+     -
+   * - `2.4. Literals <https://docs.python.org/3/reference/lexical_analysis.html#literals>`_
+     - Not Relevant
+     -
+   * - `2.4.1. String and Bytes literals <https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals>`_
+     - Partially Supported
+     - String parser doesn't support ``r"hi"``
+   * - `2.4.2. String literal concatenation <https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation>`_
+     - Supported
+     -
+   * - `2.4.3. Formatted string literals <https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals>`_
+     - Partially Supported
+     -
+   * - `2.4.4. Numeric literals <https://docs.python.org/3/reference/lexical_analysis.html#numeric-literals>`_
+     - Supported
+     - Python parses -1 as unary_neg(1) whereas we parse it as -1 directly
+   * - `2.4.5. Integer literals <https://docs.python.org/3/reference/lexical_analysis.html#integer-literals>`_
+     - Partially Supported
+     - String parser does not support binary/octal literals, big literals result in ``stoll`` error, numbers with '_' don't work in string parser, string parser interprets hex literals as 0
+   * - `2.4.6. Floating point literals <https://docs.python.org/3/reference/lexical_analysis.html#floating-point-literals>`_
+     - Partially Supported
+     - Same issues with underscores as with integer literals
+   * - `2.4.7. Imaginary literals <https://docs.python.org/3/reference/lexical_analysis.html#imaginary-literals>`_
+     - Not Supported
+     - We're interpreting these as real integers instead of imaginary numbers
+   * - `2.5. Operators <https://docs.python.org/3/reference/lexical_analysis.html#operators>`_
+     - Partially Supported
+     - Not supported: <<, >>, :=,
+   * - `2.6. Delimiters <https://docs.python.org/3/reference/lexical_analysis.html#delimiters>`_
+     - Partially Supported
+     - Not supported: **=, <<=, >>=, %=, ^=, @=, &=, //=, % for strings
+   * - `3. Data model <https://docs.python.org/3/reference/datamodel.html#>`_
+     - Not Relevant
+     -
+   * - `3.1. Objects, values and types <https://docs.python.org/3/reference/datamodel.html#objects-values-and-types>`_
+     - Not Relevant
+     - id() not supported, type() upcalls to Python
+   * - `3.2. The standard type hierarchy <https://docs.python.org/3/reference/datamodel.html#the-standard-type-hierarchy>`_
+     - Partially Supported
+     - op == with None works in Python, not supported: NotImplemented, Ellipsis, numbers.Complex, bytes, byte arrays, sets, frozen sets, generators, coroutines, async generators, modules, I/O objects, internal objects, slice objects ( though slicing is supported), classmethod
```

Comment:
I would remove `op == with None works in Python` and just list the not supported section.